investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/service/InvestigationService.java

```diff
@@ -90,10 +90,14 @@ public String processInvestigation(String value) {
                 publicHealthCaseUid = payloadNode.get("public_health_case_uid").asText();
                 investigationKey.setPublicHealthCaseUid(Long.valueOf(publicHealthCaseUid));
 
-                // Calling sp_public_health_case_fact_datamart_event
-                logger.info("Executing stored proc with ids: {} to populate PHС fact datamart", publicHealthCaseUid);
-                investigationRepository.populatePhcFact(publicHealthCaseUid);
-                logger.info("Stored proc executed");
+                try {
+                    // Calling sp_public_health_case_fact_datamart_event
+                    logger.info("Executing stored proc with ids: {} to populate PHС fact datamart", publicHealthCaseUid);
+                    investigationRepository.populatePhcFact(publicHealthCaseUid);
+                    logger.info("Stored proc executed");
+                } catch (Exception dbe) {
+                    logger.warn("Error processing PHC fact datamart: {}", dbe.getMessage());
```

Comment:
💯

Reply:
Would this also go into the DLQ?

Reply:
No, there is no topic for PHC, so we can't sent it to DLQ in that case. Otherwise entire investigation message would go to DLQ.

Reply:
In the case of an PHC datamart event erroring out, it might be a bit hard to find errors in publichealthcasefactdatamart. It is not the scope of this ticket, but I wonder if it would be more helpful to migrate it into the post-processing where the error can be put into a queue- or wherever we can keep the errors queued for this.